cli/actions/__tests__/lint.test.js

```diff
@@ -0,0 +1,39 @@
+jest.setMock('glob', {
+  sync: () => ({ length: 1 }),
+});
+jest.mock('../../logger');
+jest.mock('../../../utils/paths');
+
+describe('lint', () => {
+  global.process.exit = jest.fn();
+  const logger = require('../../logger');
+  const lint = require('../lint');
+  const eslint = require('eslint');
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('logs and exits 0 when no errors', () => {
+    lint();
+    expect(logger.end).toBeCalledWith('Your JS looks great ✨');
```

Comment:
We could but then it would look something like this:

```js
expect(logger.end.mock.calls[0][0]).toMatch(/Your JS looks great/)
```

Which would be potentially even more brittle depending on if `logger.end` was called anywhere else (`.mock.calls[x][x]` would be variable).

I think this is probably okay for now and if it ever fails we can relax this assertion or remove it entirely and assert something more useful.